.coveragerc

```diff
@@ -7,6 +7,12 @@ omit =
     homeassistant/scripts/*.py
 
     # omit pieces of code that rely on external devices being present
+
+    homeassistant/components/aqara/binary_sensor.py
+    homeassistant/components/aqara/base.py
+    homeassistant/components/aqara/__init__.py
+    homeassistant/components/aqara/util.py
+    
     homeassistant/components/acer_projector/*
     homeassistant/components/actiontec/const.py
```

Comment:
Please put these in the list with the others (below) and keep it sorted in alphabet)

Reply:
ok，updated